account_banking_mandate/views/account_banking_mandate_view.xml

```diff
@@ -68,11 +68,15 @@
 		<field name="model">account.banking.mandate</field>
 		<field name="arch" type="xml">
 			<search string="Search Banking Mandates">

```

Comment:
Thanks for the improvements! Odd spacing here, as in the other XML. Are tabs involved, in the original or new code? Maybe this is a good time to reformat on indents of 4 spaces?